tpc/pregen/media.dsl/dslplatform/Guards.java

```diff
@@ -0,0 +1,15 @@
+package serializers.dslplatform;
+
+public final class Guards {
+    /// No-oped for perf reasons
```

Comment:
This is not related with serialization, but more with using it instead of POJOs. Its fine to revert it... since its only few ms diff in create. Null cheks are done during deserialization.